cms/djangoapps/contentstore/tasks.py

```diff
@@ -185,7 +185,7 @@ def async_migrate_transcript(self, course_key, **kwargs):   # pylint: disable=un
 
             sub_tasks = []
             video_location = unicode(video.location)
-            for lang in all_transcripts.keys():
+            for lang in all_transcripts:
```

Comment:
why this changed from `.keys()`?

Reply:
It was showing up as a new warning for some reason: 
https://build.testeng.edx.org/job/edx-platform-quality-flow-pr/5858/warnings52Result/new/

Reply:
Ok, got it.